docs/_specs/spec-053-evidence-hallucination-detection.md

```
@@ -0,0 +1,413 @@
+
+
+class EvidenceGroundingError(ValueError):
+    """Raised when extracted evidence cannot be grounded in the transcript."""
+
+
+def validate_evidence_grounding(
+    evidence: dict[str, list[str]],
+    transcript_text: str,
+    *,
+    mode: str = "substring",
+    fuzzy_threshold: float = 0.85,
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)